cmd/kogito/command/shared/operatorhub.go

```diff
@@ -30,10 +30,9 @@ import (
 )
 
 const (
-	defaultOperatorPackageName   = "kogito-operator"
-	defaultOperatorChannelName   = "alpha"
-	communityOperatorSource      = "community-operators"
-	operatorMarketplaceNamespace = "openshift-marketplace"
+	DefaultOperatorPackageName   = "kogito-operator"
```

Comment:
why expose those?

Reply:
I exposed them because I need to use them in kogito_operator_test.go file under install package

Reply:
Instead, you could create a `GetOperator(...)` in the `*_test` file in this package for the tests on `install` package to use. 
We shouldn't expose those constants just for testing.

Reply:
I am not clear how to do it, could you please elaborate it more